modin/pandas/test/dataframe/test_indexing.py

```diff
@@ -1707,9 +1708,28 @@ def test_xs():
         "locomotion": ["walks", "walks", "flies", "walks"],
     }
     df = pd.DataFrame(data=d)
+
+    # to make several partitions
+    df = pd.concat([df, df], axis=0)
+    df = df.join(df, rsuffix="_y")
+
     df = df.set_index(["class", "animal", "locomotion"])
-    with warns_that_defaulting_to_pandas():
-        df.xs("mammal")
+
+    result = df.xs("mammal")
+    expected = df._to_pandas().xs("mammal")
```

Comment:
I would prefer to call exactly the method of pandas instead of converting a modin df to pandas.

Reply:
will change

Reply:
done